31412 MainProcess 2024-04-18 11:17:03,967
Traceback (most recent call last):
  File "C:\Users\GM001010686\AppData\Local\Programs\Python\Python311\Lib\site-packages\pyarmor\cli\__main__.py", line 757, in main
    main_entry(sys.argv[1:])
  File "C:\Users\GM001010686\AppData\Local\Programs\Python\Python311\Lib\site-packages\pyarmor\cli\__main__.py", line 745, in main_entry
    return args.func(ctx, args)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\GM001010686\AppData\Local\Programs\Python\Python311\Lib\site-packages\pyarmor\cli\__main__.py", line 237, in cmd_gen
    builder.process(options)
  File "C:\Users\GM001010686\AppData\Local\Programs\Python\Python311\Lib\site-packages\pyarmor\cli\generate.py", line 164, in process
    self._obfuscate_scripts()
  File "C:\Users\GM001010686\AppData\Local\Programs\Python\Python311\Lib\site-packages\pyarmor\cli\generate.py", line 122, in _obfuscate_scripts
    code = Pytransform3.generate_obfuscated_script(self.ctx, r)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\GM001010686\AppData\Local\Programs\Python\Python311\Lib\site-packages\pyarmor\cli\core\__init__.py", line 106, in generate_obfuscated_script
    return m.generate_obfuscated_script(ctx, res)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<maker>", line 702, in generate_obfuscated_script
  File "C:\Users\GM001010686\AppData\Local\Programs\Python\Python311\Lib\site-packages\pyarmor\cli\__init__.py", line 16, in process
    return meth(self, res, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<maker>", line 532, in process
  File "<maker>", line 538, in coserialize
  File "<maker>", line 583, in _build_ast_body
RuntimeError: out of license
